docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Nexus zkVM SDK Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
+## Reproducible Proving and Verification
+
+<Steps>
+
+This example demonstrates how to use the Nexus SDK to generate proving and verification keys, compile a guest program, prove its execution, and verify the proof. The example uses the `guest` package as an example guest program.
+
+The `src/main.rs` at host side is as follows:
```

Comment:
Are we following the file layout generated by SDK quick start?

Do we run this command from the SDK quick start?
```
cargo install --git https://github.com/nexus-xyz/nexus-zkvm cargo-nexus --tag 'v0.2.1'
```
Do we do something else?

Reply:
Maybe

> The `src/main.rs` at host side is as follows:

could be

> The host program `src/main.rs` is:

Reply:
@yoichi-nexus  I think there is a page dedicated for that so I don't mention it here. 
@sjudson Updated in the latest commit.

Reply:
> I think there is a page dedicated for that

Please add a link to that page. That makes the benchmark more easily reproducible.

Reply:
updated in in c8ed3c3  at the first section

Reply:
The instruction seems incomplete. I left a comment in Vercel. I'm closing this thread.